Source/PluginProcessor.cpp

```diff
@@ -261,9 +307,27 @@ void PinkTromboneAudioProcessor::processBlock (AudioBuffer<float>& buffer, MidiB
 	}
 }
 
-void PinkTromboneAudioProcessor::applyConstrictionEnvelope(float sampleVal)
+void PinkTromboneAudioProcessor::applyEnvelope(float sampleVal)

```

Comment:
If you followed my advice with the other comment, this function becomes much simpler as well